test/wanda_web/views/v1/catalog_view_test.exs

```diff
@@ -9,16 +9,43 @@ defmodule WandaWeb.V1.CatalogViewTest do
   describe "CatalogView" do
     test "renders catalog.json" do
       checks = [
-        build(:check, expectations: build_list(3, :catalog_expectation, type: :expect)),
-        build(:check, expectations: build_list(3, :catalog_expectation, type: :expect_same))
+        build(:check, expectations: build_list(2, :catalog_expectation, type: :expect)),
+        build(:check, expectations: build_list(2, :catalog_expectation, type: :expect_same))
       ]
 
+      updated_checks =
```

Comment:
Reusing here the code to format the response of the view for v1,v2. 
Maybe there is a better solution for this.
Open for suggestions.

Reply:
```
adapted_checks =
  Enum.map(%{expectations: expectations} = check ->
    %{check | expectations: Enum.map(expectations, &(&Map.drop(&1, [:warning_message])})
```